build.gradle

```diff
@@ -11,10 +12,25 @@ repositories {
     mavenCentral()
 }
 
+checkstyle {
+    toolVersion = '8.1'
+}
+
+configurations {
+    cucumberRuntime {
+        extendsFrom testRuntime
+    }
+}
+
 dependencies {
     // Test engine
     testCompile group: 'junit', name: 'junit', version: '4.12'
 
+    // Cucumber
+    testCompile group: 'info.cukes', name: 'cucumber-java', version: '1.2.5'
+    testCompile group: 'info.cukes', name: 'cucumber-picocontainer', version: '1.2.5'
+    testCompile group: 'info.cukes', name: 'cucumber-testng', version: '1.2.5'
+
```

Comment:
please add and replace that dependencies by:
 compile group: 'io.github.bonigarcia', name: 'webdrivermanager', version: '3.8.0'
    testCompile 'io.cucumber:cucumber-java:5.4.0'
    testImplementation 'io.cucumber:cucumber-java:5.4.0'
    testCompile group: 'io.cucumber', name: 'cucumber-picocontainer', version: '5.4.0'
    compile group: 'io.cucumber', name: 'cucumber-testng', version: '5.4.0'